PluginDirectories/1/wunderlist.bundle/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "categories" : [
+    "Utilities"
+  ],
+  "displayName" : "Task to Wunderlist",
+  "name" : "Wunderlist",
```

Comment:
for future reference, the `name` value should be the same as the `.bundle` file's name (minus the extension), including capitalization. I'll fix it